Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeFormUIModel.swift

```diff
@@ -119,14 +119,17 @@ class CreateSafeFormUIModel {
     }
 
     var isCreateEnabled: Bool {
-        state == .ready &&
-        name != nil && !name!.isEmpty &&
+        (state == .ready || (state == .keyNotFound && userSelectedPaymentMethod == .Relayer)) &&
+        name != nil &&
+        !name!.isEmpty &&
         chain != nil &&
         !owners.isEmpty &&
-        threshold > 0 && threshold <= owners.count &&
-        selectedKey != nil &&
+        threshold > 0 &&
+        threshold <= owners.count &&
+        (selectedKey != nil || userSelectedPaymentMethod == .Relayer) &&
```

Comment:
I think we can rewrite those conditions in order to avoid many checks for "userSelectedPaymentMethod == .Relayer"

Reply:
We probably could. But wouldn't that make it less readable? I find it more readable that the user selection overrides certain conditions.